### PR TITLE
Tail Regained checks for owner and species + deleted unused mood_event datums

### PR DESCRIPTION
**Pull request by larentoun**
## About The Pull Request

Changed how tail regaining works.

* If it's your tail, an infinite debuff is replaced with a timed one (new, on live there is no debuff)
* If it's not your tail but of same species, I guess it works, but we are more sad (new)
* If it's not your tail AND of different species, we are horrified (live)

Also a little clearing up of unused mood datums

## Why It's Good For The Game
Refactors good? Having your tail back good?

## Changelog
:cl:
balance: Now regaining a tail has three states. If you got your tail back - you'll get a temporarily debuff in exchange of infinite "tail missing" debuff (new). If it's NOT your tail and of same species - this temporarily debuff will be a bit stronger (new). If it's NOT your tail and NOT of same species - it will be a considerably stronger debuff (currently on live).
/:cl:

### CODE REVIEW

**Comment by GPeckman** [comment]
Generally speaking, if players won't see the changes then you don't include them in the changelog.

File: code/datums/mood_events/generic_negative_events.dm [hunk 1]
```diff
@@ -380,11 +340,6 @@
 	description = "I feel off-balance without my tail."
 	mood_change = -2
 
-/datum/mood_event/tail_regained_right
-	description = "My tail is back, but that was traumatic..."
-	mood_change = -2
-	timeout = 5 MINUTES
```
Comment (MrMelbert) [inline code/datums/mood_events/generic_negative_events.dm:386]: This is definitely supposed to be used so the fact that it's not probably means the event associated is bugged

Comment (MrMelbert) [reply code/datums/mood_events/generic_negative_events.dm:386]: In other words I would rather fix the reason this is unused than delete it

**Comment by MrMelbert** [comment]
Calling this a fix because it's kind of how it was originally meant to be

**Pull request merged**